plasma_framework/test/src/exits/payment/PaymentExitGameReentrantProtected.test.js

```diff
@@ -2,6 +2,7 @@ const PaymentChallengeStandardExit = artifacts.require('PaymentChallengeStandard
 const PaymentProcessStandardExit = artifacts.require('PaymentProcessStandardExit');
 const PaymentStandardExitRouter = artifacts.require('PaymentStandardExitRouterMock');
 const PaymentInFlightExitRouter = artifacts.require('PaymentInFlightExitRouterMock');
+const PaymentExitGame = artifacts.require('PaymentExitGame');
```

Comment:
changes needed to swap public `boot` functions in `PaymentStandardExitRouter ` and `PaymentInFlightExitRouter` to internal.